contracts/bonding/BondingManager.sol

```diff
@@ -995,21 +1031,24 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     }
 
     /**
-     * @dev Return LivepeerToken interface
+     * @notice Return LivepeerToken interface
+     * @return Livepeer token contract binding
```

Comment:
I think "Livepeer token contract registered with Controller" could be clearer. Same comment applies for the function comments below.

Reply:
fixed in b79bb67